docs/javascript/v2/release-notes/release-notes.mdx

```diff
@@ -12,6 +12,15 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/hms-video-store | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-store.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-store) |
 | @100mslive/react-sdk       | [![npm version](https://badge.fury.io/js/%40100mslive%2Freact-sdk.svg)](https://badge.fury.io/js/%40100mslive%2Freact-sdk)             |
 
+## 2022-05-09
+
+Released: `@100mslive/hms-video-store@0.2.99`, `@100mslive/react-sdk@0.0.16`, `@100mslive/hms-video-react@0.3.127`
+
+-   Upgrade to React 18
+-   Fix server-side subscribe degradation
```

Comment:
Should this be metioned as a fix or feature? This might be the first time it is mentioned.